### PR TITLE
Move api.controller to feature apps

### PR DESCRIPTION
**Pull request by tyom**
Addresses #74 and #78.

### CODE REVIEW

File: src/apps/api/router.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const router = require('express').Router()
+const { postcodeLookupHandler } = require('./api.controller')
+
+router
+  .route('/postcodelookup/:postcode')
+  .get(postcodeLookupHandler)
```
Comment (ztolley) [inline src/apps/api/router.js]: I like this but then isn't this different from the syntax used in PR 255,

router.get('/myaccount', getHandler)

Can we pick one style and all stick with it?

Comment (ztolley) [inline src/apps/api/router.js]: nvm  I see we use a mix of the two, especially in contacts

Comment (teneightfive) [inline src/apps/api/router.js]: The pattern I stick to is if there's only one method per route, use shorthand:

```js
router.get('/route', handler)
```

but if there's multiple methods for a route, use this syntax:

```js
router
  .route('/route')
  .get(getHandler)
  .post(postHandler)
```

Based on that style, we should probably use shorthand here.

Comment (ztolley) [inline src/apps/api/router.js]: Yup I'm following that

Comment (tyom) [inline src/apps/api/router.js]: Updated.

**Pull request merged**